dpe/src/commands/derive_context.rs

```diff
@@ -457,6 +456,19 @@ mod tests {
         svn: 0,
     };
 
+    impl Default for DeriveContextCmd {
+        fn default() -> Self {
+            Self {
+                handle: ContextHandle::default(),
+                data: [0; DPE_PROFILE.tci_size()],
+                flags: DeriveContextFlags(0),
+                tci_type: 0,
+                target_locality: 0,
+                svn: 0,
+            }
+        }
+    }
```

Comment:
Why put it in the test module? Could this be useful elsewhere?

Not a huge deal but curious

Reply:
That's a good point. I put it there because we weren't using elsewhere, but there is no reason to stop us from using it in the future. Move to non-test code.